### PR TITLE
add documentation for how to use FHIR extension in Synapse sync agent

### PR DESCRIPTION
**Pull request by RuiyiC**

### CODE REVIEW

**Comment by QuanWanxx** [comment]
Maybe need to update the deployment **image** and **Parameter/Description** for customized schema in [Step 1](https://github.com/microsoft/FHIR-Analytics-Pipelines/blob/main/FhirToDataLake/docs/Deployment.md#1-deploy-the-pipeline)?

Also need to update the script **syntax** and **Parameter/Description** in [Step 6](https://github.com/microsoft/FHIR-Analytics-Pipelines/blob/main/FhirToDataLake/docs/Deployment.md#1-deploy-the-pipeline)

File: FhirToDataLake/docs/Process FHIR extensions.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    },
+    "required": [ "id" ]
+}
+```
+
+**Note**:
+1.	The JSON schema files must be saved at **Schema** directory in the image.
+	 
+	 ![image](./assets/LiquidDirectory.png)
+
```
Comment (QuanWanxx) [inline FhirToDataLake/docs/Process FHIR extensions.md:55]: I missed a requirement, the JSON schema files need to have suffix **".schema.json"**

File: FhirToDataLake/docs/Process FHIR extensions.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ ![image](./assets/ExampleParquetData2.png)
+
+## Query customized data in Synapse Serverless SQL pool
+
+### 1.	Run the Powershell script to create the tables and views with customized schema image
+
+Browse to the scripts folder under this path (..\FhirToDataLake\scripts).
+
+Run the following PowerShell script.
+./Set-SynapseEnvironment.ps1 -SynapseWorkspaceName "Synapse name" -Storage   "Storage name" -CustomizedSchemaImage "Schema image reference"
```
Comment (QuanWanxx) [inline FhirToDataLake/docs/Process FHIR extensions.md:94]: Maybe make the example as a code content by adding ``` around it.

```
./Set-SynapseEnvironment.ps1 -SynapseWorkspaceName "Synapse name" -Storage "Storage name" -CustomizedSchemaImage "Schema image reference"
```

File: FhirToDataLake/docs/Process FHIR extensions.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+_Example:_
+./Set-SynapseEnvironment.ps1 -SynapseWorkspaceName quwansynapsews0830  -Storage fhrtosynaps6mocz2elwbwg6  -CustomizedSchemaImage exampleacr.azurecr.io/customizedtemplate:extensiontemplates
```
Comment (QuanWanxx) [inline FhirToDataLake/docs/Process FHIR extensions.md:97]: Also here, maybe make the example as a code content by adding ``` around it.

File: FhirToDataLake/docs/Process FHIR extensions.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+## Generate parquet data for FHIR extensions
+
+### 1.	Prepare the customized schema templates
+For each resource types, e.g. Patient, Observation, you need to prepare a **liquid template** and a related **JSON schema** file.
+- The liquid template will be used to convert the raw FHIR JSON data into target structure.
+- The JSON schema will be used to generate the Parquet schema and create table definitions.
+
+Follow the links for more information about [liquid template](http://dotliquidmarkup.org/) and [JSON schema](https://json-schema.org/learn/getting-started-step-by-step).
+
+Example liquid template and JSON schema for “Patient” resource, flatten the “birthplace” extension for analytics.
```
Comment (QuanWanxx) [inline FhirToDataLake/docs/Process FHIR extensions.md:14]: Below examples are intend for [Patient birthPlace](https://build.fhir.org/extension-patient-birthplace.html) extension, maybe add the reference for it.

File: FhirToDataLake/docs/Process FHIR extensions.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+# Process FHIR extensions  
+
+ ![image](./assets/ProcessExtensions.png)
+
+## Generate parquet data for FHIR extensions
+
+### 1.	Prepare the customized schema templates
+For each resource types, e.g. Patient, Observation, you need to prepare a **liquid template** and a related **JSON schema** file.
+- The liquid template will be used to convert the raw FHIR JSON data into target structure.
```
Comment (moria97) [inline FhirToDataLake/docs/Process FHIR extensions.md:9]: @QuanWanxx Shall we mention the liquid template here is using the same syntax from FHIR-Converter repo?

Comment (QuanWanxx) [reply FhirToDataLake/docs/Process FHIR extensions.md:9]: Good idea, I just found this [Filters-and-Tags](https://github.com/microsoft/FHIR-Converter/blob/main/docs/Filters-and-Tags.md), maybe we can add the reference for that?

**Review by QuanWanxx — APPROVED** [review]

File: FhirToDataLake/docs/Deployment.md [hunk 6]
```diff
@@ -53,16 +53,21 @@ This solution enables you to query against the entire FHIR data with tools such
...
     | Group Id (new) | For data filtering use. If the Filter scope is set as `Group`, you need to fill the group Id, otherwise leave it blank.|
     | Required Types (new) | For data filtering use. Specify which types of resources will be included. For example, _type=Patient_ would return only patient resources. All resource types will be exported if not specified. Leave it blank if no filter is applied.|
     | Type Filters (new) | For data filtering use. Use along with the requiredTypes configuration. The value is a comma-separated list of FHIR queries that further restrict the results. All data of requiredTypes will be exported if not specified. Leave it blank if no filter is applied.|
     | Package url | The build package of the agent. You need not change this. |
-    | App Insight Location | You can find logs in the deployed application insight resource. You need not change this. |
+    | Deploy App Insights | Whether to deploy the Application Insights. You can find logs in the deployed application insight resource. You need not change this. |
+    | App Insight Location | The location to deploy the App Insight. You need not change this. |
```
Comment (moria97) [inline FhirToDataLake/docs/Deployment.md:64]: Maybe this setting can be changed to choose a location that has quota for app insights? @QuanWanxx

**Review by moria97 — APPROVED** [review]

**Pull request merged**